Assets/Scripts/Manager/ThemeMusicManager.cs

```diff
@@ -23,16 +27,22 @@ public class ThemeMusicManager : MusicManager {
 			Debug.LogError("Multiple instances of ThemeMusicManager!");
 		}
 		instance = this;
-	}
 
-	void Start(){
+		initAudioSource ();
+
 		String currentLevel = SceneManager.GetActiveScene ().name;
 		if (mainThemeScenes.ToList ().Contains (currentLevel)) {
 			startMainTheme ();
 		} else if (mechanicalThemeScenes.ToList ().Contains (currentLevel)) {
 			startMechanicalTheme ();
 		}
 	}
+		
+	void initAudioSource(){
+		AudioSource[] source = GetComponentsInChildren<AudioSource> ();
```

Comment:
@jayfriso nope, but don't think it will be too hard for us to keep it in order, and they are all using the same prefab.  Fell free to fix this if you have a better idea, I'm just giving a quick fix